libraries/standard/mqtt/include/mqtt.h

```diff
@@ -383,6 +567,30 @@ MQTTStatus_t MQTT_Disconnect( MQTTContext_t * pContext );
  * #MQTTIllegalState if an incoming QoS 1/2 publish or ack causes an
  * invalid transition for the internal state machine;
  * #MQTTSuccess on success.
+ * 
+ * <b>Example</b>
+ * @code{c}
+ * 
+ * // Variables used in this example.
+ * MQTTStatus_t status;
+ * uint32_t timeoutMs = 100;
+ * // This context is assumed to be initialized and connected.
+ * MQTTContext_t * pContext;
+ * 
+ * while( true )
+ * {
+ *      status = MQTT_ProcessLoop( pContext, timeoutMs );
+ * 
+ *      if( status != MQTTSuccess )
+ *      {
+ *          // Determine the error. It's possible we might need to disconnect TCP.
```

Comment:
It's possible we might need to disconnect TCP --> It's possible we might need to disconnect the underlying transport connection.

Reply:
Thanks, will update the wording in next PR